packages/plugins/plugin-assistant/src/components/Chat/Chat.tsx

```diff
@@ -351,18 +358,17 @@ const ChatPrompt = ({
         onSubmit={handleSubmit}
       />
 
-      <div />
-      <ChatReferences
-        classNames='col-span-2 flex pis-1 items-center'
-        space={space}
-        context={processor.context}
-        onUpdate={handleUpdateReferences}
-      />
-
       <ChatOptions
+        space={space}
         blueprintRegistry={processor.blueprintRegistry}
-        context={processor.context}
-        onUpdateBlueprint={onUpdateBlueprint}
+        context={
+          /* TODO(thure): Why are we prop-drilling `context` and `blueprintRegistry`? If it’s a context can we not just `useContext`? */ processor.context
```

Comment:
I think just because these concepts are rather in flux and shift around a lot. There's still a question as to how the binding happens and whether or not it should have access to the whole registry at that time (its fine when registry is static but maybe less so if its open/dynamic)